go/tools/gopackagesdriver/bazel_json_builder.go

```diff
@@ -17,80 +17,119 @@ package main
 import (
 	"context"
 	"fmt"
+	"path/filepath"
 	"strings"
 )
 
 type BazelJSONBuilder struct {
-	bazel      *Bazel
-	query      string
-	tagFilters string
-	targets    []string
+	bazel    *Bazel
+	requests []string
 }
 
 const (
-	OutputGroupDriverJSONFile = "go_pkg_driver_json_file"
-	OutputGroupStdLibJSONFile = "go_pkg_driver_stdlib_json_file"
-	OutputGroupExportFile     = "go_pkg_driver_export_file"
+	RulesGoStdlibLabel = "@io_bazel_rules_go//:stdlib"
 )
 
-func NewBazelJSONBuilder(bazel *Bazel, query, tagFilters string, targets []string) (*BazelJSONBuilder, error) {
+func (b *BazelJSONBuilder) fileQuery(filename string) string {
+	if filepath.IsAbs(filename) {
+		fp, _ := filepath.Rel(b.bazel.WorkspaceRoot(), filename)
+		filename = fp
+	}
+	return fmt.Sprintf(`kind("go_library", same_pkg_direct_rdeps("%s"))`, filename)
```

Comment:
This may return empty result when the `filename` is a test file like `foo_test.go`, because no `go_library` would depend on such file

Reply:
yeah, test files are not done atm, if you have an idea, i'm all ears